src/ModelContextProtocol/Client/McpClientExtensions.cs

```diff
@@ -24,7 +24,7 @@ public static Task PingAsync(this IMcpClient client, CancellationToken cancellat
         return client.SendRequestAsync(
             RequestMethods.Ping, 
             parameters: null,
-            McpJsonUtilities.JsonContext.Default.Object,
+            McpJsonUtilities.JsonContext.Default.Object!,
```

Comment:
Was this warning before?

Reply:
This probably answers my question https://github.com/modelcontextprotocol/csharp-sdk/pull/195/files#r2026782132